src/Evolution/Systems/GeneralizedHarmonic/Constraints.hpp

```diff
@@ -144,4 +144,38 @@ void two_index_constraint(
     const tnsr::iaa<DataType, SpatialDim, Frame>&
         three_index_constraint) noexcept;
 // @}
+
+// @{
+/*!
+ * \brief Computes the generalized-harmonic 4-index constraint.
+ *
+ * \details Computes the independent components of the generalized-harmonic
+ * 4-index constraint. The constraint itself is given by Eq. (45) of
+ * https://arXiv.org/abs/gr-qc/0512093v3,
+ * \f{eqnarray}{
+ * C_{ijab} = 2 \partial_{[i}\Phi_{j]ab},
+ * \f}
+ * where \f$\Phi_{iab} = \partial_i\psi_{ab}\f$. Because the constraint is
+ * antisymmetric on the two spatial indices, here we compute and store
+ * only the independent components of \f$C_{ijab}\f$. Specifically, we
+ * compute
+ * \f{eqnarray}{
+ * D_{iab} \equiv \frac{1}{2} \epsilon_{ijk} C_{ijab}
```

Comment:
index balance and up/down style on the levi-cevita. Also mention that the \epsilon is being raised and lowered with the kronecker delta.